### DISCUSSION TITLE
Issues discovered -

### DISCUSSION THREAD

**Discussion by whxuezhe** [Alex313031/Thorium-Win-AVX2#59] [Q&A]
Cannot run thorium build installer on Core 2 i3 2120 machine, incompatible program. The version used is 109.0.5362.0, and there is no problem with Firefox 110.0a1 of a friend; is it going to be a New Year's Eve to build a new version?

**Comment by gz83** [comment]
> Cannot run thorium build installer on Core 2 i3 2120 machine, incompatible program. The version used is 109.0.5362.0, and there is no problem with Firefox 110.0a1 of a friend; is it going to be a New Year's Eve to build a new version?

@whxuezhe Your processor does not support the avx2 instruction set, please use the avx version.
